### PR TITLE
Hopefully finally fix droplimb()

### LINKED ISSUE #6616: People not dying after losing their chest post-necrosis
(WEB REPORT BY: kammerjunk REMOTE: 158.69.55.2:7777)
>Revision (Should be above if you're viewing this from ingame!)
2148960ff95e0433e0312997bc750d3da263a152

>General description of the issue
A patient lost his chest after a Necrosis virus. After giving him his missing *limbs* back (left leg, right foot, left foot), I woke him up and he was alive and unharmed.

>What you expected to happen
I expected the patient to die and his limbs to explode off the body.

>What actually happened
The patient turned out fine. I don't know if he could have put his suit back on after taking it off.

>Steps to reproduce if possible
Get Necrosis and don't die? I'm not sure.

### PR DESCRIPTION
**Pull request by ghost**
Eat Genetic Power : 
- You can no longer eat groins (which caused extremely serious technical problems because that limb cannot actually be amputated). Now you'll be told that it is inappropriate (this is a christian server after all) and will return
- Please stop using usr for fucks sakes
- Remove t_his because it isn't actually needed. Tweak m_his to be compatible with neutral in case a robot somehow gets that genetic power
- Always use limb.display_name when limbs are cast in text

Necrosis : 
- Remove a lot of blankspace and empty indents
- Now only works on humans because organs are a human thing (at least properly, casting a mob that you haven't checked as human is shitcode of the highest order)
- Basically formatting

Organ System : 
- droplimp() will now flat out return from the start if it is passed a UPPER_TORSO or LOWER_TORSO external organ (Read : Torso and groin exclusively) instead of applying a flag, even if overridden. This breaks shit bad otherwise

Fixes #6616 and a bunch of organ gibbing issues that were reported or not

### CODE REVIEW

**Comment by Probe1** [comment]
What's wrong with ```usr``` ?  The PR seems fine though.

**Comment by dylanstrategie** [comment]
If you really want people to "eat each other whole" with the Eat power, the code infrastructure already exists and a single conditional in the code needs to be checked

The code itself is hit-and-miss and I'm already trying to fix it for Xenos

**Comment by 9600bauds** [comment]
We vorestation now
